ext/libyuv.cmd

```diff
@@ -22,6 +22,6 @@ git checkout 464c51a0
 mkdir build
 cd build
 
-cmake -G Ninja -DCMAKE_BUILD_TYPE=Release ..
+cmake -G Ninja -DCMAKE_BUILD_TYPE=Release -DCMAKE_POSITION_INDEPENDENT_CODE=ON -DBUILD_SHARED_LIBS=OFF ..
```

Comment:
Frankie: We need to make sure all the build scripts in the `ext` directory compile the respective libraries with position-independent code enabled. This pull request only made such changes to the libyuv scripts. Have you checked the scripts for the other libraries?

Reply:
I've been building libavif as a shared library with static dependencies for a while in [pillow-avif-plugin](https://github.com/fdintino/pillow-avif-plugin), and I knew this to be a problem with libyuv. It looks like dav1d, rav1e, SVT-AV1, and libsharpyuv (by way of libwebp) all pass `-fPIC` when being built statically. libgav1, avm, and aom appear not to (and in fact, it looks like I'm also passing in `-fPIC` into aom; I don't link against libgav1 or avm). Should I add `-DCMAKE_POSITION_INDEPENDENT=ON` to all of the `.cmd` scripts, or just the ones that don't already have it as a default for static builds?

Reply:
We only need to add `-DCMAKE_POSITION_INDEPENDENT=ON` to the `.cmd` scripts that don't already have it as a default for static builds. But it may be time-consuming to find out which `.cmd` scripts need `-DCMAKE_POSITION_INDEPENDENT=ON`. So I think it is fine to add it to all the `.cmd` scripts.

Reply:
Last I checked, most dealt properly with PIC. We can just run the CI and only modify the ones who fail. Just to keep the .cmd minimal

Reply:
In any event, I've already identified which don't deal with PIC (libgav1, avm, aom, and libyuv). I had added it to all based on the initial feedback, but I'll change it back to only those scripts.